nevergrad/benchmark/experiments.py

```diff
@@ -1053,15 +1053,15 @@ def neuro_control_problem(seed: tp.Optional[int] = None) -> tp.Iterator[Experime
                             
                             
 @registry.register
-def simpletsp(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+def simpletsp(seed: tp.Optional[int] = None, complex_tsp: bool = False) -> tp.Iterator[Experiment]:
```

Comment:
it would be more readable to rename this `simple_tsp` btw (and complex_tsp)

Reply:
Done.